chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -20,7 +20,8 @@
 ################################
 ## Airflow Scheduler Deployment/StatefulSet
 #################################
-
+# Are we using a scheduler in k8s?
```

Comment:
This comment raises a question, can we instead have a comment saying: `Condition for cases where we run only workers in K8s`

Reply:
It honestly may be best to just remove the comment. I think its pretty obvious what the condition checks for, regardless of why the user chooses it.

Reply:
@amoghrajesh, what do you think about just getting rid of this comment instead?

Reply:
Go for it @kajanajustin , it makes sense to me not to have it at all.

Reply:
@amoghrajesh kk getting rid of it